eclipse-scout-core/src/dataobject/serialize/DataObjectDeserializer.ts

```diff
@@ -76,14 +79,20 @@ export class DataObjectDeserializer implements DataObjectDeserializerModel, Obje
 export interface DataObjectDeserializerModel extends ObjectModel<DataObjectDeserializer> {
   /**
    * Controls the kind of object that will be created when deserializing unknown DataObjects.
-   * If true, a pojo will be created for unknown DOs. If false, instances of {@link BaseDoEntity} will be created.
-   * Default is false.
+   * If <code>true</code>, a pojo will be created for unknown DOs. If <code>false</code>, instances of {@link BaseDoEntity} will be created.
+   * Default is <code>false</code>.
    *
    * A DataObject can be unknown e.g. if:
    * <ol>
    *   <li>The _type attribute of the object to deserialize cannot be found in TS.</li>
    *   <li>There is no _type attribute and the TypeScript attribute declaration (metadata) does not provide a class type which could be used as fallback.</li>
    * </ol>
    */
-  createPojoIfDoIsUnknown: boolean;
+  createPojoIfDoIsUnknown?: boolean;
+  /**
+   * Controls whether the '_typeVersion' property is removed or not when deserializing DataObjects.
+   * If <code>true</code>, the '_typeVersion' will be set on the deserialized DataObject. If <code>false</code>, it will be ignored.
+   * Default is <code>false</code> if {@link createPojoIfDoIsUnknown} is <code>false</code> and <code>true</code> otherwise.
```

Comment:
Also use backticks here instead of &lt;code>?
And you can remove the "or not" part: wheter .. is removed when ...